src/component/Home/PetCard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import whiteCat from '../../images/whiteCat.jpg';
```

Comment:
Looks like the whiteCat.jpg image is missing and leads to an error when rendering the component.